tests/eko/test_output_struct.py

```diff
@@ -130,7 +130,9 @@ class TestEKO:
     def _default_cards(self):
         t = tc.generate(0, 1.0)
         o = oc.generate([10.0])
-        return compatibility.update(t, o)
+        nt, no = compatibility.update(t, o)
+        no["rotations"]["pids"] = no["rotations"]["targetpids"]
```

Comment:
Shouldn't we put this in `compatibility.update`?

Reply:
This I don't know, I believe it is something very specific to this test. Most of the case we do not need this because `[rotations][pids]` will be already there

Reply:
Ok, then maybe upgrade default cards, such that we don't need it also here :)

In the end, default cards are for the users (as everything else in `ekobox`), we don't want to generate incompatible ones :)